# true: localization mode, false: mapping mode
bool localization_mode
---
bool success